README.md

```diff
@@ -91,8 +92,8 @@ The signature base string is a concatenated string generated from the following
 * `Authorization-Header-Parameters`: normalized parameters similar to section 9.1.1 of OAuth 1.0a.  The parameters are the id, nonce, realm, and version from the Authorization header. Parameters are sorted by name and separated by '&' with name and value separated by =, percent encoded (urlencoded)
 * `Added Signed Headers`: The normalized header names and values specified in the headers parameter of the Authorization header. Names should be lower-cased, sorted by name, separated from value by a colon and the value followed by a newline so each extra header is on its own line.
 * `Timestamp`:  The value of the X-Authorization-Timestamp header
-* `Content-Type`: The lowercase value of the "Content-type" header (or empty string if absent). Omit for a GET or HEAD request.
-* `Body-Hash`: The base64 encoded SHA-256 digest of the raw body of the HTTP request, for POST, PUT, PATCH, DELETE or other requests that may have a body. Omit for GET or HEAD. This should be identical to the string sent as the X-Authorization-Content-SHA256 header.
+* `Content-Type`: The lowercase value of the "Content-type" header (or empty string if absent). Omit if Content-Length == 0.
```

Comment:
Given that this is text, not code, I might rather say "Omit if Content-Length is 0" or "Omit if Content-Length equals 0"